server/src/internalClusterTest/java/org/opensearch/explain/ExplainActionIT.java

```diff
@@ -305,4 +308,25 @@ public void testStreamExplain() throws Exception {
         result = Lucene.readExplanation(esBuffer);
         assertThat(exp.toString(), equalTo(result.toString()));
     }
+
+    public void testQueryRewrite() {
+        client().admin()
+            .indices()
+            .prepareCreate("twitter")
+            .setMapping("user", "type=integer", "followers", "type=integer")
+            .setSettings(Settings.builder().put(SETTING_NUMBER_OF_SHARDS, 2).put("index.number_of_routing_shards", 2))
```

Comment:
(Style Nit) It's strange to see one setting as a constant and the other one not.  I think you can use `INDEX_NUMBER_OF_ROUTING_SHARDS_SETTING` here.